pjmedia/include/pjmedia/av_sync.h

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (C) 2025 Teluu Inc. (http://www.teluu.com)
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+#ifndef __PJMEDIA_AV_SYNC_H__
+#define __PJMEDIA_AV_SYNC_H__
+
+ /**
+  * @file av_sync.h
+  * @brief Inter-media Synchronization.
+  */
+#include <pjmedia/types.h>
+
+
+PJ_BEGIN_DECL
+
+
+/**
+ * @defgroup PJMEDIA_AV_SYNC Inter-media Synchronization
+ * @ingroup PJMEDIA_SESSION
+ * @brief Synchronize presentation time of multiple media in a session.
+ * @{
+ *
+ * A call session may consist of multiple media, e.g: some audio and some
+ * video, which frequently have different delays when presented in the
+ * receiver side. This module synchronizes all media in the same session
+ * based on NTP timestamp & RTP timestamp info provided by the sender in
+ * RTCP SR.
+ *
+ * Here are steps to use this module:
+ * 1. Create AV sync using #pjmedia_av_sync_create().
+ * 2. Adds all media to be synchronized using #pjmedia_av_sync_add_media().
+ * 3. Call #pjmedia_av_sync_update_ref() each time the media receiving
+ *    an RTCP SR packet.
+ * 4. Call #pjmedia_av_sync_update_pts() each time the media returning
+ *    a frame to be presented, e.g: via port.get_frame(). See more about
+ *    this function below.
+ * 5. Call #pjmedia_av_sync_del_media() when a media is removed from the
+ *    session.
+ * 6. Call #pjmedia_av_sync_destroy() when the session is ended.
+ * 
+ * More about #pjmedia_av_sync_update_pts():
+ * - The function will return non-zero when delay adjustment in the media is
+ *   needed for synchronization. When it returns a positive number, the media
+ *   should increase the delay as many as the returned number, in millisecond.
+ *   When it returns a negative number, the media should try to decrease
+ *   the delay as many as the returned number, in millisecond.
+ *   For example in audio stream, the delay adjustment can managed using
+ *   #pjmedia_jbuf_set_min_delay().
+ * - The function will keep track on the delay adjustment progress, for
+ *   example, after some delay adjustment are requested to a media and
```

Comment:
a little bit too vague.
how many are "some" delay adjustment? how much Is relatively high?
maybe provide the constants/settings that will decide this.